service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java

```diff
@@ -359,6 +372,105 @@ public OrderWithAddressesResponseDto saveCurrentAddressForOrder(OrderAddressDtoR
         return findAllAddressesForCurrentOrder(uuid);
     }
 
+    private OrderAddressDtoRequest geoCodeSearchingRequest(String searchRequest) {
+        List<Locale> locales = List.of(new Locale("uk"), new Locale("en"));
+        GeoApiContext context = new GeoApiContext.Builder().apiKey(googleApiKey).build();
```

Comment:
you can define `GeoApiContext` as a bean and inject it whenever needed. (instead of creating new instance for each request)